extensions/wikia/FounderEmails/events/FounderEmailsDaysPassedEvent.class.php

```diff
@@ -79,13 +79,13 @@ private function getEmailController( $activateDay ) {
 	}
 
 	public static function register( $wikiParams, $debugMode = false ) {
-		global $wgFounderEmailsEvents;
+		global $wgFounderEmailsEvents, $wgCityId;
 
 		$founderEmailObj = FounderEmails::getInstance();
 
 		$wikiFounder = $founderEmailObj->getWikiFounder();
-		$wikiFounder->setLocalPreference( "founderemails-joins", true );
-		$wikiFounder->setLocalPreference( "founderemails-edits", true );
+		$wikiFounder->setLocalPreference( "founderemails-joins", true, $wgCityId );
+		$wikiFounder->setLocalPreference( "founderemails-edits", true, $wgCityId );
```

Comment:
[It is](https://github.com/Wikia/app/blob/SERVICES-783/lib/Wikia/src/Domain/User/Preferences/LocalPreference.php#L15), that's how we found this :P